components/org.wso2.carbon.identity.organization.resource.hierarchy.traverse.service/src/main/java/org/wso2/carbon/identity/organization/resource/hierarchy/traverse/service/exception/OrgResourceHierarchyTraverseClientException.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.organization.resource.hierarchy.traverse.service.exception;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Exception class for client-side errors during organization resource hierarchy traversal.
+ * <p>
+ * This class handles exceptions where client-side errors occur, capturing error codes, messages, and descriptions
+ * to provide detailed context for troubleshooting.
+ */
+public class OrgResourceHierarchyTraverseClientException extends OrgResourceHierarchyTraverseException {
+
+    private String[] messages;
+
+    /**
+     * Constructs a new exception with an array of specified error messages.
+     *
+     * @param messages Detailed error messages
```

Comment:
```suggestion
     * @param messages Detailed error messages.
```
Some other places also have missing fullstop.
Better to format all

Reply:
https://github.com/wso2-extensions/identity-organization-management/pull/415/commits/938dc308459c20feb1fa02b2116a22ba884a80be